src/ResourceManager/AzureBackup/Commands.AzureBackup/Cmdlets/Jobs/GetAzureRMBackupJob.cs

```diff
@@ -139,10 +140,12 @@ protected override void ProcessRecord()
                 WriteDebug(String.Format(Resources.TypeFilter, Type));
                 WriteDebug(String.Format(Resources.JobIdFilter, JobId));
 
+                DateTimeFormatInfo format = new CultureInfo("en-US").DateTimeFormat;
                 Mgmt.CSMJobQueryObject queryParams = new Mgmt.CSMJobQueryObject()
                 {
-                    StartTime = From.Value.ToString("yyyy-MM-dd hh:mm:ss tt"),
-                    EndTime = To.Value.ToString("yyyy-MM-dd hh:mm:ss tt"),
+
+                    StartTime = From.Value.ToString("yyyy-MM-dd hh:mm:ss tt", format),
+                    EndTime = To.Value.ToString("yyyy-MM-dd hh:mm:ss tt", format),
```

Comment:
So the REST API expects the times to be in this en-US format, which implies that the portal code has something similar?